tox.ini

```diff
@@ -0,0 +1,14 @@
+[tox]
+isolated_build = True
+envlist =
+    py{310}
```

Comment:
Let's try to be compatible with Python 3.7+ (all the currently supported python versions)